CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-### master
-
```

Comment:
FYI - this should not be removed

Reply:
lol, that's from starting at the qui-gon branch. :shrug: I'll put it back I guess.